app/handlers/moderator.py

```diff
@@ -341,9 +342,10 @@ async def approve_report_handler(
             bot=bot,
         )
         await callback_query.message.edit_text(
-            "<b>{reporter}</b> получил <b>+{reward_amount}</b> кармы в награду за репорт!".format(
+            "<b>{reporter}</b> получил <b>+{reward_amount}</b> кармы в награду за репорт!{admin_url}".format(
```

Comment:
Let's remove this exclamation mark :)
```suggestion
            "<b>{reporter}</b> получил <b>+{reward_amount}</b> кармы в награду за репорт{admin_url}".format(
```

Reply:
removed